bootstrap-themes/src/main/java/de/agilecoders/wicket/themes/markup/html/italia/ItaliaCssReference.java

```diff
@@ -0,0 +1,27 @@
+package de.agilecoders.wicket.themes.markup.html.italia;
+
+import org.apache.wicket.request.resource.CssResourceReference;
+
+public class ItaliaCssReference extends CssResourceReference {
+	
+	private static final long serialVersionUID = 4361205879317055824L;
+
+    public ItaliaCssReference() {
+        super(ItaliaCssReference.class, "css/bootstrap-italia.min.css");
```

Comment:
No need to have `.min` in the file name. Wicket will use the `.css` file in DEVELOPMENT mode and the `.min.css` in DEPLOYMENT mode